environment.yml

```diff
@@ -29,6 +29,7 @@ dependencies:
 - sphinxext-opengraph
 - sphinx-notfound-page
 - sphinx-design
+- watermark
```

Comment:
This is already above at line 19 from Alex's PR

Reply:
haha good catch!